Craigslist/Craigslist.Selenium/Content/PagedList.css

```diff
@@ -0,0 +1,166 @@
+.pagination {
+  display: inline-block;
+  padding-left: 0;
```

Comment:
The same css in Selenium project. It definitely has nothing to do with UI test automation!